src/main/java/com/faforever/client/chat/AbstractChatTabController.java

```diff
@@ -687,4 +725,36 @@ protected void onDisplay() {
   protected void onHide() {
 
   }
+
+  public Button getEmoticonsButton() {
+    return emoticonsButton;
+  }
```

Comment:
This getter is unnecessary nothing uses it and I don't think we want to expose this button anyway outside of as necessary for the fxml